app/models/reports/bulk_recommend_report.rb

```diff
@@ -94,7 +94,7 @@ def csv_row(trainee)
       row = [
         trainee_report.trn,
         trainee_report.provider_trainee_id,
-        trainee_report.hesa_id&.gsub(/(.*)/, "'\\1"), # prevent Excel from converting it to scientific notation
+        trainee_report.hesa_id,
```

Comment:
I think we're doing this substitution twice and doing it wrong.
- First is in https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/models/reports/trainee_report.rb#L405 and second time is this line.
- We're doing it at start and end of string, rather than just start, so this might help:
`.gsub(/^(.*)/, "'\\1")`

That's why we're getting `''1234567890''`

Reply:
@tomtrentham good spot, I think you're right. Have a look at [this commit](https://github.com/DFE-Digital/register-trainee-teachers/pull/4405/commits/757a46b563f8835edab3f43b17a908aac248aa35) and let me know what you think. I've put the CsvValueSanitiser back and updated the `hesa_id` method on the trainee_report. I've removed the call to gsub in `bulk_recommend_report` since the report is already doing it as you say.